tests/foreman/cli/test_scparam.py

```diff
@@ -20,7 +20,7 @@ def run_puppet_module(self):
 
     @run_only_on('sat')
     @tier1
-    def test_bugzilla_1047794(self):
+    def test_positive_list(self):
         """@Test: Check if SmartClass Paramter Info generates an error
 
         @Feature: SmartClass Paramter - Info
```

Comment:
I think `@Test` and `@Feature` description needs to be updated with `List` instead of `Info`